### PR TITLE
SO-5013: Fix show admin conditional rendering

### PR DESCRIPTION
**Pull request by jondauz**
https://teamsnap.atlassian.net/browse/SO-5013

### CODE REVIEW

File: src/js/components/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -87,14 +87,16 @@ const Toolbar = ({
...
         </div>
-        <div>
-          <Button type="link" style={btnStyle} onClick={() => setActiveMenu(Menu.Admin)}>
-            {showAdmin && <Icon className="Icon u-spaceLeftLg" name="wrench" style={iconStyle} />}
-          </Button>
-          {activeMenu === Menu.Admin && (
-            <div className="u-posAbsolute Toolbar-flyout">{adminBody}</div>
-          )}
-        </div>
+        {showAdmin && (
```
Comment (BillyFigueroa) [inline src/js/components/Toolbar/Toolbar.tsx:90]: Might need to check if it is a ghost inspector user also?

Comment (jondauz) [reply src/js/components/Toolbar/Toolbar.tsx:90]: How do I check for ghost inspector user?

Comment (BillyFigueroa) [reply src/js/components/Toolbar/Toolbar.tsx:90]: @jondauz hmmm never mind, this is something you can check in teamsnap spas

**Review by BillyFigueroa — APPROVED** [review]

File: src/js/components/Toolbar/Toolbar.tsx (hunk 1 above)
Comment (bradcypert) [reply src/js/components/Toolbar/Toolbar.tsx:90]: Yeah, as this is open source and available to people outside if TeamSnap, we don't want to check for anything specific to what we use at TeamSnap. If a conditional needs to be available to a component where we need data like that, it needs to be available via a prop, which it is with the showAdmin prop. In fact, the implementation we have currently might even be too specific, but we can worry about that another time.

**Pull request merged**